### PR TITLE
fix: Incorrect field focused in property pane when empty binding is present in one of fields

### LINKED ISSUE #28844: [Task]: Show a learning popover post one-click binding
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

Show a learning popover that prompts users to switch to JS mode when one-click-binding is complete.

### PR DESCRIPTION
**Pull request by arunvjn**
#### PR fixes following issue(s)
Fixes #28844 
>
#### Type of change
- Chore (housekeeping or task changes that don't impact user perception)
- Experimental
>
>
#### How Has This Been Tested?
- [x] Manual
>
>
#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/AssistiveBinding/assistiveBinding_spec.ts [hunk 1]
```diff
@@ -139,12 +139,12 @@ describe("Assistive Binding", function () {
...
 
-  it("6. Works correctly when user toggles JS", () => {
-    entityExplorer.SelectEntityByName("Button1", "Widgets");
-    propPane.ToggleJSMode("onClick");
-    propPane.ValidatePropertyFieldValue("onClick", "{{}}");
-    propPane.TypeTextIntoField("onClick", "hello", false);
-    // This asserts that cursor was originally between the curly braces after the user switched to js mode
-    propPane.ValidatePropertyFieldValue("onClick", "{{hello}}");
-  });
+  // it("6. Works correctly when user toggles JS", () => {
```
Comment (sbalaji1192) [inline app/client/cypress/e2e/Regression/ClientSide/AssistiveBinding/assistiveBinding_spec.ts:142]: not needed anymore?

Comment (arunvjn) [reply app/client/cypress/e2e/Regression/ClientSide/AssistiveBinding/assistiveBinding_spec.ts]: Uncommented it

**Pull request closed without merging**